src/perf/lib/Tcp.cpp

```diff
@@ -199,9 +199,22 @@ TcpWorker::~TcpWorker()
     CxPlatEventUninitialize(WakeEvent);
 }
 
-bool TcpWorker::Initialize(TcpEngine* _Engine)
+bool TcpWorker::Initialize(TcpEngine* _Engine, uint16_t AssignedCPU)
 {
     Engine = _Engine;
+    ExecutionContext.Callback = DoWork;
+    ExecutionContext.Context = this;
+    ExecutionContext.Ready = TRUE;
+    ExecutionContext.NextTimeUs = 0; // TODO: Figure out why adding this line makes lowlat tcp work?
```

Comment:
You need to default this to zero to indicate you don't have a timer you're waiting on.